client/js/heat.js

```diff
@@ -0,0 +1,91 @@
+class HeatMapRow {
+  /**
+   * format for a row for heat map values obj
+   * @param {number[]} data The array of 0s and 1s for a row in a heat map
+   */
+  constructor(data) {
+    this.data = data;
+  }
+}
+
+// data will be from backend next time
```

Comment:
nit: since you know this is something that needs to change in the future, you should use an official TODO comment like

`// TODO: retrieve data from the backend instead of using a hardcoded value`